metricbeat/docs/modules/aws/ec2.asciidoc

```diff
@@ -0,0 +1,23 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[metricbeat-metricset-aws-ec2]]
+=== aws ec2 metricset
+
+experimental[]
```

Comment:
Umm, it appears as experimental but the module is beta, in any case I guess this is not related to this PR //cc @kaiyan-sheng

Reply:
@jsoriano @sayden Yeah I had a PR changing aws module from experimental to beta. Do we need a rebase here?

Reply:
I think (but I'm not sure) there's no need to rebase as the docs generation scripts will do their job once it's merged